lgc/CMakeLists.txt

```diff
@@ -32,67 +32,59 @@ add_dependencies(LLVMlgc
 )
 
 ### Cached Project Options #############################################################################################
-option(LLPC_BUILD_RENOIR   "LLPC support for Renoir?"   ON)
+option(LLPC_BUILD_RENOIR  "LLPC support for Renoir?"    ON)
```

Comment:
@Flakebi , why did you remove a space here? is there any coding standard about this? if so,  why not remove one space between "LLPC support for Renoir?" and "ON"?

Reply:
I wanted to align it with the line below that I inserted, that’s all.
I guess there is no standard about the amount of spaces in there, it varies greatly. The only system behind it that I see is that lines that are near each other are aligned to each other.

Reply:
got it.not a problem then.